azure-mvn-settings.xml

```diff
@@ -1,34 +1,10 @@
 <settings>
   <mirrors>
     <mirror>
-      <id>jboss-public-repository-group</id>
-      <mirrorOf>*,!quarkus-nexus-release,!quarkus-nexus-snapshot</mirrorOf>
-      <name>jboss</name>
-      <url>http://repository.jboss.org/nexus/content/groups/developer/</url>
+      <id>google-maven-central</id>
+      <name>GCS Maven Central mirror EU</name>
+      <url>https://maven-central-eu.storage-download.googleapis.com/repos/central/data/</url>
+      <mirrorOf>central</mirrorOf>
```

Comment:
I'm pretty sure you need to keep the JBoss Nexus repository as well?

Reply:
No. It's declared in the pom themselves as we could not remove it from there (see the root pom.xml).

Reply:
Sure, yes I'm aware of the mention in the pom.xml.

We still _need_ the JBoss Nexus repo though. Not saying that your change is breaking, but it would be cleared to have the server id made explicit here. But as you wish, it was meant as a minor comment.

Reply:
BTW why the EU mirror? is our CI running in EU?

If that's the case you might be better of with the official Maven mirror for europe:

   <mirror>
      <id>UK</id>
      <name>UK Central</name>
      <url>http://uk.maven.org/maven2</url>
      <mirrorOf>central</mirrorOf>
    </mirror>

I didn't suggest that as I was assuming the Azure pipeline was running in US.

Reply:
The reason why I removed it from there is that it doesn't break anything to remove it as it's declared in the parent pom and I would have had to redeclare Maven central here to be sure it's taken first.

In any case, we will go the Stuart's way on this and have a dedicated mirror set up on Azure.

This PR is here as a backup plan if we have any issue.

Reply:
@Sanne so I asked Stuart and he said our instances are in West Europe. That's why I chose this mirror (and it's indeed faster than the US based one).

About the UK mirror, I'm a bit worried that they might have limitations similar to Maven Central itself.

Reply:
:+1: